src/test/java/bio/terra/service/filedata/google/firestore/EncodeFixture.java

```diff
@@ -125,6 +125,12 @@ public SetupResult setupEncode(
         SnapshotSummaryModel snapshotSummary = dataRepoFixtures.createSnapshot(
             custodian, datasetSummary.getName(), profileId, "encodefiletest-snapshot.json");
 
+        dataRepoFixtures.addSnapshotPolicyMember(
+            custodian,
+            snapshotSummary.getId(),
+            IamRole.STEWARD,
+            steward.getEmail());
+
         // TODO: Fix use of IamProviderInterface - see DR-494
         dataRepoFixtures.addSnapshotPolicyMember(
```

Comment:
Do we still need this now that the custodian is added as snapshot steward?

Reply:
right, exactly.  Just making sure different users can grant the access